frontend/src/views/welcome/Login.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <AppRow>
+    <AppCol class="col3">
+      <section id="menu">
+        <ul>
+          <li><a href="#">What is new MSP Direct?</a></li>
+          <li><router-link :to="{ name: 'CredentialsInfo' }">What are new log-in Credentials</router-link></li>
+          <li><a href="https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/system-access/msp-direct" target="_blank">Transactions in MSP Direct?</a></li>
+          <li><a href="https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/identity-and-authentication-services" target="_blank">Get BC Services Card or BCeID</a></li>
+        </ul>
+      </section>
```

Comment:
BCGov has a bit of a standard styling side menus e.g.)
https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/system-access/msp-direct
https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/system-access/msp-direct

Maybe we could create this and add it to the design system?

Reply:
> BCGov has a bit of a standard styling side menus e.g.) https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/system-access/msp-direct https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/system-access/msp-direct
> 
> Maybe we could create this and add it to the design system?

Thanks for pointing this out. I'll update the styling.

Reply:
> It might also be nice to include a link back to the Login page in this box. We can bring that up with the Business. Having to click welcome to get back to login wasn't intuitive to me.

The box only shows on the Login page. Are you talking about navigating back from to the Login page from Credentials Info? If so maybe a "Return to Login" or "Back to Login" button at the bottom of the page?

Reply:
Updated menu to Gov standards and included menu on Credentials page which will allow for access to Login.